pkgs/applications/radio/gnuradio/default.nix

```
@@ -292,6 +293,12 @@ stdenv.mkDerivation (finalAttrs: (shared // {
   patches = [
     # Not accepted upstream, see https://github.com/gnuradio/gnuradio/pull/5227
     ./modtool-newmod-permissions.patch
+    # https://github.com/gnuradio/gnuradio/issues/7458
+    (fetchpatch {
+      name = "gnuradio-numpy_2-compatibility.patch";
+      url = "https://github.com/gnuradio/gnuradio/commit/4be7ac7b5095a4c0d394456934102b8666b7c68c.patch";
```

```suggestion
      url = "https://github.com/gnuradio/gnuradio/commit/8fbc5eb4b7214a4cb029ccae97205a85d49bdd48.patch";
```
This is the merged commit, in case the PR branch is ever deleted.
The hash should be the same but I didn't check.

LGTM otherwise

---

You are right, thanks for the refinement and for explaining. Fixed it and will merge when CI is green enough.